--- bugzilla-5.0.6/Bugzilla/Constants.pm	2019-04-18 03:44:10.000000000 +0200
+++ bugzilla-5.0.6-nix/Bugzilla/Constants.pm	2024-07-26 23:00:44.791514611 +0200
@@ -700,7 +700,7 @@
     $datadir     = "data";
   }
 
-  $datadir = "$libpath/$datadir";
+  $datadir = ($ENV{NIX_BZ_DATADIR} // die "You must set NIX_BZ_DATADIR to point to your datadir");
 
   # We have to return absolute paths for mod_perl.
   # That means that if you modify these paths, they must be absolute paths.
@@ -714,7 +714,7 @@
     'templatedir'    => "$libpath/template",
     'template_cache' => "$datadir/template",
     'project'        => $project,
-    'localconfig'    => "$libpath/$localconfig",
+    'localconfig'    => ($ENV{NIX_BZ_LOCALCONFIG} // die "You must set NIX_BZ_LOCALCONFIG to your localconfig file"),
     'datadir'        => $datadir,
     'attachdir'      => "$datadir/attachments",
     'skinsdir'       => "$libpath/skins",
